Resources/Views/base.leaf

```diff
@@ -9,7 +9,7 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap" rel="stylesheet">
     
-    <title>#import("title")</title>
+    <title>SwiftLeeds - The Unique iOS Swift Conference in the UK</title>
```

Comment:
This should be injected using the title part of the other leaf template, not hard-coded

Reply:
Because it's a single-page website I thought we'd want the same title across the whole website?

Reply:
That mechanism is already there by passing in title, it just makes it easier to expand.  Eventually I assume we'll want more details about past conferences, i.e. agendas, so there will be sublemental pages that don't fit into the main Home page, that was my thinking

Reply:
No problem @pdamonkey updated now 👍🏼